openapi3.yaml

```diff
@@ -1,57 +1,143 @@
 openapi: 3.0.1
 info:
-  title: service-name
-  description: basic template for map colonies service
+  title: products catalog service
+  description: catalog service
   version: 1.0.0
   license:
     name: MIT
     url: https://opensource.org/licenses/MIT
-
+servers:
+  - url: http://localhost:8080
 paths:
-  /anotherResource:
+  /getProducts:
     get:
-      operationId: getAnotherResource
+      operationId: getProducts
       tags:
-        - anotherResource
-      summary: gets the resource
+        - getProducts
+      summary: get products from db catalog, accroding to the filter in requestBody
+      requestBody:
```

Comment:
Why is there a `body` in this `get` operation?

Reply:
In order to get from user how he wants to filter the products catalog table. for example if he wants all the products with `minZoom>2` and `maxZoom<8` he can enter that in the `body` according to the format i defined.